File: WS_U3_AppDiplomas/Director.php
<?php

class Director {
    private $idDirector;
    private $grado;
    private $firma;
    private $nombre;
    private $Usuario_idUsuario;

    public function __construct() {
    }

    public function __construct2($grado, $firma, $nombre, $Usuario_idUsuario) {
        $this->grado = $grado;
        $this->firma = $firma;
        $this->nombre = $nombre;
        $this->Usuario_idUsuario = $Usuario_idUsuario;
    }

    public function __construct3($idDirector, $grado, $firma, $nombre, $Usuario_idUsuario) {
        $this->idDirector = $idDirector;
        $this->grado = $grado;
        $this->firma = $firma;
        $this->nombre = $nombre;
        $this->Usuario_idUsuario = $Usuario_idUsuario;
    }

    public function getIdDirector() {
        return $this->idDirector;
    }

    public function setIdDirector($idDirector) {
        $this->idDirector = $idDirector;
    }

    public function getGrado() {
        return $this->grado;
    }

    public function setGrado($grado) {
        $this->grado = $grado;
    }

    public function getFirma() {
        return $this->firma;
    }

    public function setFirma($firma) {
        $this->firma = $firma;
    }

    public function getNombre() {
        return $this->nombre;
    }

    public function setNombre($nombre) {
        $this->nombre = $nombre;
    }

    public function getUsuario_idUsuario() {
        return $this->Usuario_idUsuario;
    }

    public function setUsuario_idUsuario($Usuario_idUsuario) {
        $this->Usuario_idUsuario = $Usuario_idUsuario;
    }

    public function __toString() {
        return "Director{" . "idDirector=" . $this->idDirector . ", grado=" . $this->grado . ", firma=" . $this->firma . ", nombre=" . $this->nombre . ", Usuario_idUsuario=" . $this->Usuario_idUsuario . '}';
    }
}


?>

File: TAP_U3ProyectoFinalDali/src/main/java/com/mycompany/domain/Director.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.domain;

/**
 *
 * @author charl
 */
public class Director {
    private int idDirector;
    private String grado;
    private byte[] firma;
    private String nombre;
    private int Usuario_idUsuario;

    public Director() {
    }

    public Director(String grado, byte[] firma, String nombre, int Usuario_idUsuario) {
        this.grado = grado;
        this.firma = firma;
        this.nombre = nombre;
        this.Usuario_idUsuario = Usuario_idUsuario;
    }

    public Director(int idDirector, String grado, byte[] firma, String nombre, int Usuario_idUsuario) {
        this.idDirector = idDirector;
        this.grado = grado;
        this.firma = firma;
        this.nombre = nombre;
        this.Usuario_idUsuario = Usuario_idUsuario;
    }

    public int getIdDirector() {
        return idDirector;
    }

    public void setIdDirector(int idDirector) {
        this.idDirector = idDirector;
    }

    public String getGrado() {
        return grado;
    }

    public void setGrado(String grado) {
        this.grado = grado;
    }

    public byte[] getFirma() {
        return firma;
    }

    public void setFirma(byte[] firma) {
        this.firma = firma;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getUsuario_idUsuario() {
        return Usuario_idUsuario;
    }

    public void setUsuario_idUsuario(int Usuario_idUsuario) {
        this.Usuario_idUsuario = Usuario_idUsuario;
    }

    @Override
    public String toString() {
        return "Director{" + "idDirector=" + idDirector + ", grado=" + grado + ", firma=" + firma + ", nombre=" + nombre + ", Usuario_idUsuario=" + Usuario_idUsuario + '}';
    }
    
}
